frame/mosaic/Cargo.toml

```diff
@@ -0,0 +1,73 @@
+[package]
+name = "pallet-mosaic"
+version = "0.1.0"
+authors = ["Composable Developers"]
+homepage = "https://composable.finance"
+edition = "2021"
+rust-version = "1.56"
+
+[[bin]]
+name = "plot"
+path = "src/plots.rs"
```

Comment:
you can make this in test. why? one of variants is to produce docs from test docs